### PR TITLE
Feature/jmeter

### PR DESCRIPTION
**Pull request by lokeshrangineni**

### CODE REVIEW

File: automation/jmeter/runBenchmarkTest.sh [hunk 1]
```diff
@@ -2,15 +2,18 @@
...
 jmeter -n -t "${currentDir}/AMQ_Jmeter_Test_Plan_local.jmx" \
   -JproducerDuration=10 \
   -JconsumerDuration=20 \
+  -JproducerSamples=1000 \
+  -JconsumerSamples=2000 \
   -l "${outputDir}/jmeter_results_${HOST_ADDRESS}.jtl"
 
 printf  "Calculating throughput from JTL file."
-./gen_throughput.sh "-i" "${outputDir}/jmeter_results_${HOST_ADDRESS}.jtl" -a 200 -b 200 -f csv -c "JMS Subscriber" -p "JMS Publisher" > "${outputDir}/jmeter_csv_results_${HOST_ADDRESS}.csv"
+./gen_throughput.sh "-i" "${outputDir}/jmeter_results_${HOST_ADDRESS}.jtl" -a 1000 -b 2000 -f csv -c "JMS Subscriber" -p "JMS Publisher" > "${outputDir}/jmeter_csv_results_${HOST_ADDRESS}.csv"
```
Comment (sgahlot) [inline automation/jmeter/runBenchmarkTest.sh:16]: `-a` is for consumer sample size and it's value is `2000` in the jmeter jmx file but it's specified as `1000` here.
Same goes for `-b` (producer sample size) - value in jmx is `500` but it is specified as `2000` here.

**Review by sgahlot — COMMENTED** [review]
@lokeshrangineni I've left a comment for you to review.

**Review by sgahlot — COMMENTED** [review]
Another comment for you to review

File: automation/jmeter/AMQ_Jmeter_Test_Plan.jmx [hunk 2]
```diff
@@ -18,6 +18,16 @@
             <stringProp name="Argument.value">15</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
+          <elementProp name="producerSamples" elementType="Argument">
+            <stringProp name="Argument.name">producerSamples</stringProp>
+            <stringProp name="Argument.value">500</stringProp>
```
[the remark this answers is not in the document]
Comment (lokeshrangineni) [reply automation/jmeter/AMQ_Jmeter_Test_Plan.jmx:23]: This is the default value i have left when we run through GUI mode. I am overriding this value from CLI.

File: automation/jmeter/runBenchmarkTest.sh (hunk 1 above)
Comment (lokeshrangineni) [reply automation/jmeter/runBenchmarkTest.sh]: Thanks for pointing out. I have updated the PR.

**Review by sgahlot — APPROVED** [review]
Sorry, I didn't realize that you were overwriting it in the script.

**Pull request merged**